src/ostorlab/runtimes/cloud/runtime.py

```diff
@@ -483,6 +484,16 @@ def _check_agents_exist(
                 return False
         return True
 
+    def _to_serialized(self, value) -> Union[bytes, memoryview, List[Any], str]:
```

Comment:
Where is this method used?

Reply:
Apply in values of args to solve problem of Byte scalar bug :  "value": self._to_serialized(arg.value),